auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -316,11 +316,15 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
 
     this.isLoading = 1 // truthy
     this.$route.query.isNumberedCompanyRequest && await this.createNumberedBusiness(this.currentAccountSettings.id)
-    await this.syncBusinesses()
+    await this.syncBusinesses(this.enableSpGpDba)
     this.lastSyncBusinesses = Date.now()
     this.isLoading = 0 // falsy
   }
 
+  private get enableSpGpDba (): boolean {
+    return LaunchDarklyService.getFlag(LDFlags.EnableSpGpDba) || false
+  }
+
```

Comment:
Makes sense that this is a GETTER but if this is temporary means of flagging i think it might be easier to just retrieve this value in the 1 spot you are using it. 
ie `getTarget` method. 
I'm not sure the purpose of passing it as a parameter in the `syncBusinesses` method.
Maybe i'm not seeing something though.

Reply:
I agree with this -- if it's only used in `syncBusinesses()` then just declare a local variable in there.

Reply:
Fixed.